app/controllers/admin/sponsors_controller.rb

```diff
@@ -0,0 +1,51 @@
+class Admin::SponsorsController < ApplicationController
+before_action :set_sponsors
```

Comment:
I don't think you need this, because all your sponsor should only be shown in the index, right?
You won't be duplicating that line anywhere 😄